From 7ceb294d35b13e5f2674f02ad880ba53d84436e3 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 12 Jul 2023 10:25:09 +0200
Subject: [PATCH 325/841] feat(stm32mp25-fdts): add iwdg1 on eval board

Enable iwdg1 on board file.
Specify the timeout even if the driver ignores it and uses the
reset values already present in the watchdog registers.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a5b304da0603e173327d8241c78f21650d0a39a9 feat(stm32mp25-fdts): add iwdg1 on eval board]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I34c43512320655691971f085f2802b8c44372765
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/317863
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp257f-ev1.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 2386ef4bd..62ed0bea0 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -149,6 +149,11 @@
 	};
 };
 
+&iwdg1 {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &ommanager {
 	pinctrl-names = "default";
 	pinctrl-0 = <&ospi1_clk_pins_a
-- 
2.47.2

